scripts/scripts.js

```diff
@@ -1031,3 +1031,202 @@ export function lerp(start, end, amt) {
   return parseFloat(value.toFixed(decimals));
 }
+
+export function loadLottie() {
+  const BUNDLE_PATH = `${window.hlx.codeBasePath}/scripts/lottie.min.js`;
+
+  const scriptLoaded = document.querySelector('.js-lottie');
+  if (!scriptLoaded) {
+    const script = document.createElement('script');
+    script.src = BUNDLE_PATH;
+    script.classList.add('js-lottie');
+    document.body.append(script);
+  }
+}
+
+export function playLottie(player) {
+  if (player) {
+    try {
+      player.play();
+    } catch {
+      const play = () => {
+        player.play();
+        player.removeEventListener('ready', play);
+      };
+      player.addEventListener('ready', play);
+    }
+  }
+}
+
+export const collapseSection = (element) => {
+  // Get the height of the element's inner content, regardless of its actual size
+  const sectionHeight = element.scrollHeight;
+
+  // Temporarily disable all css transitions
+  const elementTransition = element.style.transition;
+  element.style.transition = '';
+
+  // Scroll back to top of parent
+  const parent = element.closest('.ops-accordion-item');
+  const top = parent.getBoundingClientRect()?.top;
+  window.scrollTo({
+    top,
+    // behavior: 'smooth',
+  });
+
+  /**
+   * On the next frame (as soon as the previous style change has taken effect),
+   * explicitly set the element's height to its current pixel height, so we
+   * aren't transitioning out of 'auto'
+   */
+  requestAnimationFrame(() => {
+    element.style.height = `${sectionHeight}px`;
+    element.style.transition = elementTransition;
+
+    /**
+     * On the next frame (as soon as the previous style change has taken effect),
+     * have the element transition to height: 0
+     */
+    requestAnimationFrame(() => {
+      element.style.height = '0px';
+    });
+  });
+
+  // Mark the section as "currently collapsed"
+  element.setAttribute('data-collapsed', 'true');
+};
+
+export const removeActive = (targetClass) => {
+  const activeButton = document.querySelector(`${targetClass} .is-active`);
+  if (activeButton) {
+    activeButton.classList.remove('is-active');
+  }
+};
+
+export const fadeOut = (element, callback, targetClass = 'is-current-route') => {
+  element.style.opacity = 0;
+
+  const transitionOut = () => {
+    element.classList.remove(targetClass);
+    element.classList.remove('stagger-transitions');
+
+    // Remove this event listener so it only gets triggered once
+    element.removeEventListener('transitionend', transitionOut);
+
+    if (callback) {
+      callback();
+    }
+  };
+
+  element.addEventListener('transitionend', transitionOut);
+};
+
+export const fadeIn = (element, targetClass = 'is-current-route') => {
+  if (!element) {
+    return;
+  }
+  element.classList.add(targetClass);
+
+  setTimeout(() => {
+    element.style.opacity = 1;
+    element.classList.add('stagger-transitions');
+  }, 100);
+};
+
+// Lottie animations for each categoiry
+const LOTTIE_PATHS = {
```

Comment:
these seem specific to the intro block, maybe we could either 
1. make `showRoute()` generic and pass the lottie path (or path map like this object) as a param
or
2. move the functions that are specific to the intro block into that block's js
wdyt?

Reply:
So I can just scope it to inside the function, intro and category both make use of it.